src/slick.groupitemmetadataprovider.ts

```diff
@@ -21,7 +21,9 @@ const Utils = IIFE_ONLY ? Slick.Utils : Utils_;
  * @constructor
  * @param inputOptions
  */
-export class SlickGroupItemMetadataProvider {
+export class SlickGroupItemMetadataProvider implements SlickPlugin {
+  pluginName = "GroupItemMetadataProvider" as const;
```

Comment:
can you change this to single quote, I will update ESLint to prefer single quote in a separate commit

Reply:
:+1:, changed & force-pushed